lib/ts/recipe/passwordless/types.ts

```diff
@@ -209,6 +243,44 @@ export type RecipeInterface = {
           }
     >;
 
+    verifyAndDeleteCode: (
```

Comment:
```suggestion
    verifyCode: (
```

Reply:
Pass a non optional flag called `deleteCode` (so this will require a core change).

We expose a `verifyAndDeleteCode` function in the index file, which calls verifyCode with deleteCode: true all the time.